meson.build

```diff
@@ -17,11 +17,23 @@ config = configuration_data()
 config.set('PREFIX', get_option('prefix'))
 configure_file(input: 'config.h.in', output: 'config.h', configuration: config)
 
-version_h = vcs_tag(
-  command: ['git', 'describe', '--tags'],
+git_version = run_command('git', 'describe', '--tags', check: false).stdout().strip()
```

Comment:
This does not work - it runs only during `meson setup`, so you will build wrong info when building locally.

vsc_tag arrange things so they run during `meson compile`, so you get up to date info on every build.

AI mode suggest to use a custom_target() that will run a program generating the header. This seems like a better way anyway, meson language is pretty bad (by design, so people will not write code in meson build files).